### ISSUE TITLE
Consider renaming Tapable -> Tappable

### ISSUE THREAD

**Issue by st-pasha** [flame-engine/flame#863]
# What could be improved
Rename mixins `Tapable` -> `Tappable` and similarly `HasTapableComponents` -> `HasTappableComponents`.

# Why should this be improved
The word "tapable" [is a misspelling](https://www.merriam-webster.com/dictionary/tapable), and it's just weird to have a class with a misspelled name.

# Any risks?
With Dart 2.13 type renames can be done [safely](https://medium.com/dartlang/announcing-dart-2-13-c6d547b57067):
```
mixin Tappable {...}
@Deprecated('Use Tappable instead')
typedef Tapable = Tappable;
```

**Comment by spydon** [comment]
Sounds good, we can have breaking changes in the RCs so we don't need the deprecation mixin. Do you want to submit a PR for this? :)

**Comment by spydon** [comment]
This is now done and will be released with `1.0.0-releasecandidate.13`
